sqlmesh/core/context.py

```diff
@@ -1153,10 +1154,9 @@ def plan_builder(
             )
             if max_interval_end_per_model:
                 default_end = max(max_interval_end_per_model.values())
-        else:
-            default_end = None
-
-        default_start = to_date(default_end) - timedelta(days=1) if default_end and is_dev else None
+                default_start = to_date(min(max_interval_end_per_model.values())) - timedelta(
```

Comment:
why don’t you need the else branch anymore?

Reply:
Because we initialize this value to None at the top. No need to do set None again.